config/routes.rb

```diff
@@ -1146,6 +1146,7 @@
         scaledown
         open_admin_ui
         open_admin_ui_done
```

Comment:
NOTE I forgot RHV had this action, I'll see if I can refactor it to fit in the native_console feature